locales/puppet.pot

```diff
@@ -0,0 +1,124 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016 Puppet, Inc.
+# This file is distributed under the same license as the Puppet automation framework package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Puppet automation framework 4.6.2-137-g151642b\n"
```

Comment:
Since that looks like the result of a `git describe` I'm assuming it does. I'll regenerate this file and see if it updates 4.8.0-something.

Side note: we may end up not even checking this file into the repo. Since the important part needed by Transifex is the PO associated with each target language, and our goal is generate those files as part of a CI job, tracking this file would probably just cause unnecessary churn.